db/upgrade.php

```diff
@@ -46,8 +46,8 @@ function facetoface_send_admin_upgrade_msg($data) {
 
     $table = new html_table();
     $table->head = array('Custom field ID',
-                         'Custom field original shortname',
-                         'Custom field new shortname');
+        'Custom field original shortname',
```

Comment:
I know this isn't really a change in this tracker. I only noticed because the indentation has been changed. However, these should be lang strings instead of hard coded.
A good idea here would be to open a new issue to fix this.

Reply:
Hi @mattporritt, 
This is fixed in another issue #74.